code/solutions/policy-as-code-with-guard/example_bucket.yaml

```diff
@@ -0,0 +1,19 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: Sample template that describes an Amazon S3 bucket (uksb-1q9p31idr)
```

Comment:
The description should be consistent with other labs, e.g. `AWS CloudFormation workshop - Helper scripts (uksb-1q9p31idr).`